projects/js-packages/publicize-components/src/components/social-image-generator/panel/index.js

```diff
@@ -83,6 +99,13 @@ const SocialImageGeneratorPanel = ( { prePublish = false } ) => {
 					/>
 					<hr />
 					<ImageOptions />
+					<hr />
+					<TemplatePicker
+						// TODO: add functionality
+						onSelect={ template => console.log( 'selected ' + template ) } // eslint-disable-line
+						value={ 'highway' }
```

Comment:
This is just a placeholder for now and will be updated as part of 1202583289493814-as-1204097909752724/f.